templates/resources/evaluating-a-data-package.md

```diff
@@ -16,8 +16,6 @@ To evaluate a data package via the [EDI Data Portal](https://portal.edirepositor
 
 From the Data Portal, click the **Login** button and enter EDI account credentials.
 
-<img class="screen-shot" src="/static/images/login-portal.png" width="85%">
```

Comment:
@kzollove Don't remove any images from this page.

Reply:
Images have been restored in the PR